node.html

```diff
@@ -0,0 +1,108 @@
+            crossorigin="anonymous"></script>
+    <link href="/css/daccs-css.css" rel="stylesheet">
+
+    <script src="/js/load_menu.js"></script>
+    <script src="/js/load_footer.js"></script>
+    <script src="/js/load_banner.js"></script>
+    <script src="/js/load_pictures.js"></script>
+    <meta charset="UTF-8">
+
+    <title>Title</title>
+</head>
+<body>
+
+<!-- Include menu -->
+<div id="menu"></div>
+<!-- end menu-->
+
+<!-- Include banner -->
+<div id="banner"></div>
+<!-- end banner -->
+
+<div class="row mt-5">
+    <div class="container">
+        <div id="page_title_div" class="col-12 text-center">
+
+        </div>
+    </div>
+</div>
+
+<div class="container mt-3">
+        <div class="row g-0 border rounded d-flex flex-column position-static overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative info-background">
+            <div class="col-12 col-sm-8 px-2">
+                    <div class="row">
+                        <div class="col-9">
+                            <!--H2 title -->
+                            <div id="name" class="mb-0"></div>
+                            <!--H3 title-->
+                            <div id="affiliation" class="mb-1 text-body-secondary"></div>
+                            <div id="description"></div>
+                            <div id="url"></div>
+                             <div>
+                                <p>Date Added: <span id="last_updated"></span></p>
+                                <p>Status: <span id="status"></span></p>
+                                <p>Contact: <span id="contact"></span></p>
+                                <p>Version: <span id="version"></span></p>
+                            </div>
+                        </div>
+
+                        <div class="col-3 pt-4">
+                             <!-- Small sharp icon next to title -->
+                             <a id="small-logo-link" target="_blank"><img id="title-icon-img"></a>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                            <div class="col-12 text-center">
+                                <h4 class="text-decoration-underline">Services Available</h4>
+                            </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-12">
+                            <div id="services" class="table-responsive"></div>
+                        </div>
+                    </div>
+            </div>
+
+            <div class="col-0 col-sm-4 d-none d-lg-block overflow-hidden shadow-lg">
+                <!--Sets logo as background image-->
+                <div id="background-icon">
+                    <!--Div used to apply blur effect to background image -->
+                    <div id="node-overlay-container" class="foreground-img-blur mb-auto">
```

Comment:
The blurred image area is supposed to disappear on small screens because the image becomes too stretched out and looks bad.  The empty rectangular space that remains is a side effect of hiding the blurred image.  It will hide the contents of the column (rectangular space) but not the column itself.

I've reduced the column size in small screen sizes but it will still be there.

Reply:
I don't love it but it's not worth spending time fixing for now